src/zrc/datamodel/models.py

```diff
@@ -239,6 +365,32 @@ def __str__(self):
         return "Status op {}".format(self.datum_status_gezet)
 
 
+class Resultaat(models.Model):
+    """
+    Het behaalde RESULTAAT is een koppeling tussen een RESULTAATTYPE en een
+    ZAAK.
+    """
+    uuid = models.UUIDField(
+        unique=True, default=uuid.uuid4,
+        help_text="Unieke resource identifier (UUID4)"
+    )
+    # relaties
+    zaak = models.ForeignKey('Zaak', on_delete=models.CASCADE)
+    resultaat_type = models.URLField()
+
+    toelichting = models.TextField(
+        max_length=1000, blank=True,
```

Comment:
we decided to drop the `max_length` from `TextField`s a while ago

Reply:
I guess to be discussed. Limiting it sounds like a good idea for validation, so it prevents people from abusing fields for huge datasets and to prevent database differences.

I'll remove for now, since we need a consistent solution overall.